06week/Checkpoint2.js

```diff
@@ -0,0 +1,105 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = = strNums.split(',').map(function(item) {
+    return parseInt(item, 10);
+});// Must split the array before turning the strings into numbers.
+
+
+// Find the sum of the even values
+const sumEvens = strNums.reduce((total, num) =>{
```

Comment:
The reduce function is a higher order function that allows flexibility in the accumulation loop.  The first two arguments of the callback give you access to the accumulator and current value in the loop.  So after you have an array to loop, you could use reduce.  However, in this function, you are not fully satisfying the build of reduce.  There are also a few syntactical errors that prevent this one from running